week1/index.js

```diff
@@ -1,10 +1,71 @@
-// To run this file: node index.js
-// Then in your browser: http://localhost:8080
-var http = require('http');
+// Declare global variables
+const http = require('http');
+//start server 
+const server = http.createServer();
+// the state which will be increased or decreased on the user request
+let state = 10; 
+//This port was already chosen by the assignment maker.
+const port = 8080;
+//Object containing the respons text and the content type.
+let responsToSendBack = {
```

Comment:
As I've commented elsewhere, the use of globals here is probably not the best approach. It's something new developers often do, including myself - on my first big programming project, I had to eventually go back and comb through a thousand lines of code to remove the dependencies on global variables.

See a very extensive description of why globals are bad here:
http://wiki.c2.com/?GlobalVariablesAreBad

What I would do instead is to pass the data back and forth through functions as arguments. This also ensures that the data will be consistent within a single request (what do you think will happen in your current code if two requests happen at the exact same time?).